superset-frontend/src/explore/components/controls/MetricsControl.jsx

```diff
@@ -238,6 +242,17 @@ class MetricsControl extends React.PureComponent {
     this.props.onChange(this.props.multi ? optionValues : optionValues[0]);
   }
 
+  moveLabel(dragIndex, hoverIndex) {
+    const { value } = this.state;
+
+    const newValues = [...value];
+    [newValues[hoverIndex], newValues[dragIndex]] = [
+      newValues[dragIndex],
+      newValues[hoverIndex],
+    ];
+    this.setState({ value: newValues });
+  }
```

Comment:
Not a blocker, but maybe we can abstract `AdhocFilterControl` and `MetricsControl` into a shared `AdhocOptionsControl` (hope someone can come up a better name than me).